tgui/packages/tgui/interfaces/PreferencesMenu/CharacterPreferences/MainPage.tsx

```diff
@@ -512,15 +512,7 @@ export function MainPage(props: MainPageProps) {
 
   const mainFeatures = [
     ...Object.entries(data.character_preferences.clothing),
-    ...Object.entries(data.character_preferences.features).filter(
-      ([featureName]) => {
-        if (!currentSpeciesData) {
-          return false;
-        }
-
-        return currentSpeciesData.enabled_features.indexOf(featureName) !== -1;
-      },
-    ),
```

Comment:
A lot of the primary features never set a `relevant_x` or `relevant_y` (as you can see in the rest of the pr

So theoretically ethereal color, moth wings, lizard parts etc. were all accessible to all species - they just weren't visible from being filtered here